src/uct/sm/mm/base/mm_md.h

```diff
@@ -61,7 +61,7 @@ typedef struct uct_mm_md {
 
 
 /* Check if available on current machine */
-typedef ucs_status_t (*uct_mm_mapper_query_func_t)();
+typedef ucs_status_t (*uct_mm_mapper_query_func_t)(int *attach_shm_file_p);
```

Comment:
pls describe return value in attach_shm_file_p

Reply:
done